src/webview/MessageList.js

```diff
@@ -180,13 +198,78 @@ class MessageList extends Component<Props> {
       showMessagePlaceholders,
     });
 
+    /**
+     * Effective URL of the MessageList webview.
+     *
+     * We provide all HTML at creation (or refresh) time; the document named
+     * here is not loaded, and doesn't even need to exist. It serves only as a
+     * placeholder, so that relative URLs and cross-domain security restrictions
+     * have somewhere to believe that this document originates from.
+     */
+    const baseUrl = `${assetsPath}/index.html`;
+
+    /**
+     * Paranoia^WSecurity: only load `baseUrl`, and only load it once. Any other
+     * requests should be handed off to the OS, not loaded inside the WebView.
+     */
```

Comment:
Nit: I like to reserve the `/**` jsdoc marker for comments that live up to being documentation. Here's a nice description of the principle, in just a slightly different context:
https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#use--for-public-quality-private-documentation
(That style guide in general is pretty great, and a lot of especially the documentation section generalizes quite well.)

This comment is a fine one here -- this isn't an interface that needs documentation -- but should just be marked with plain `/*` or `//`.